### DISCUSSION TITLE
How to handle connection error? (follow up issue #423)

### DISCUSSION THREAD

**Discussion by marco-minervino** [nornir-automation/nornir#706] [Q&A] [answered]
Hi,
There was any follow up to [this question](https://github.com/nornir-automation/nornir/issues/423)?
Unfortunately the discourse group is not even more active.

Here I'm trying to understand if I can build a list of unreachable devices while i run my script, to track those i had connection problem with.

Thank you.

**Comment by dbarrosop** [comment, marked as answer]
> Here I'm trying to understand if I can build a list of unreachable devices while i run my script, to track those i had connection problem with.

The nornir object already has such list, please, check the following tutorial https://nornir.readthedocs.io/en/latest/tutorial/failed_tasks.html#

In any case, if you want to do it yourself you can either use a grouped task with your own logic or a Processor
